sample/06-mongoose/src/cats/cats.controller.ts

```diff
@@ -16,4 +16,14 @@ export class CatsController {
   async findAll(): Promise<Cat[]> {
     return this.catsService.findAll();
   }
+
+  @Get(':id')
+  async findOne(@Param("id") _id: string): Promise<Cat> {
```

Comment:
Can we rename it to "id" instead?
```suggestion
  async findOne(@Param("id") id: string): Promise<Cat> {
```
Similarly in the method below and in the corresponding service class